src/ploneintranet/search/interfaces.py

```diff
@@ -0,0 +1,61 @@
+""" Interfaces for search API
+"""
+from zope.interface import Interface, Attribute
+
+from . import _
+
+
+class IGlobalSearch(Interface):
```

Comment:
Yeah that's what I wanted but @wengole said no. @adamcheasley you have the deciding vote.